### PR TITLE
[FIX-4] - Styles and carousel throttle time

### PR DESCRIPTION
**Pull request by IrynaMukhina**

### CODE REVIEW

File: src/app/components/auth/auth.scss [hunk 1]
```diff
@@ -69,22 +64,29 @@ button {
...
   }
 
+  button {
+    border: none;
+    font-size: 1rem;
+  }
+}
+
 .sign-up__container,
 .sign-up__container {
```
Comment (TorskyM) [inline src/app/components/auth/auth.scss]: Remove unnecessary class declaration

File: src/app/components/product-cart-item/product-cart-item.scss [hunk 2]
```diff
@@ -4,13 +4,13 @@ $item-shadow-color: rgba(100, 100, 100, .4);
...
-  grid-template-columns: 140px 1fr 25% 15% 10% 10%;
+  grid-template-columns: 120px 1fr 25% 15% 10% 10%;
   grid-auto-rows: 25%;
   grid-gap: 10px;
   position: relative;
-  width: 100%;
-  padding: 25px 0;
   border-top: 1px solid $item-shadow-color;
+  padding: 20px 0;
+  width: 100%;
```
Comment (TorskyM) [inline src/app/components/product-cart-item/product-cart-item.scss]: Format by style order

File: src/app/shared/product-carousel/product-carousel.component.ts [hunk 3]
```diff
@@ -1,10 +1,15 @@
 import {
   Component, OnInit, AfterViewInit, AfterViewChecked,
-  ViewChild, HostListener, ElementRef, Input, ChangeDetectorRef
+  ViewChild, HostListener, ElementRef, Input, ChangeDetectorRef, OnDestroy
```
Comment (TorskyM) [inline src/app/shared/product-carousel/product-carousel.component.ts]: Align imports

File: src/app/shared/product-carousel/product-carousel.component.ts [hunk 4]
```diff
@@ -72,19 +82,27 @@ export class ProductCarouselComponent implements OnInit, AfterViewInit, AfterVie
...
+        : this.moveToPrev();
     }
 
     this.containerScroll = this.productsContainer.nativeElement.scrollLeft;
 
     this.toggleButtonsState();
   }
 
-  constructor(private cd: ChangeDetectorRef) {}
+  constructor(private cd: ChangeDetectorRef) { }
```
Comment (TorskyM) [inline src/app/shared/product-carousel/product-carousel.component.ts]: private readonly

File: src/app/shared/product-carousel/product-carousel.html [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <div class="carousel__wrapper content__container">
   <div class="carousel">
-    <button (click)="moveToPrev()" class="left" [disabled]="!canMoveToPrev">
+    <button class="prev" [disabled]="!canMoveToPrev" #prevButton>
       <fa-icon [icon]="faArrowLeft" class="arrow-icon"></fa-icon>
```
Comment (andriiveremchuk) [inline src/app/shared/product-carousel/product-carousel.html]: Use ordering

```*structuralDirective
origin-attribute
[binding]
(method)```

Check everywhere

**Comment by IrynaMukhina** [comment]
@TorskyM and @andriiveremchuk  thank you both for reviewing.

**Pull request merged**